### DISCUSSION TITLE
Email is assigned to the wrong school

### DISCUSSION THREAD

**Discussion by Shoes01** [community/community#33036] [GitHub Education] [answered]
I am applying for Teacher status, but the application process automatically assigns me the wrong school. Faculty emails in the province all end with @nbed.nb.ca, which is ambigious, but the system decided I work a specific school. Sadly for me: the wrong one!

What are the steps I must take to change which school GitHub thinks I work at? 

I have searched for similar discussions, and the few that I have found seem to be fixed individually (https://github.com/orgs/community/discussions/17732)

Thanks.

**Comment by jfullstackdev** [comment, marked as answer]
posting here might help because Staff members are here, but there is no assurance

you may want to try the Support Contact, 

https://support.github.com/contact/account

**Reply by Shoes01** [reply]
Thank you! I have done that and will mark this solved.
